WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -532,6 +532,57 @@ - (WPMediaRequestID)imageWithSize:(CGSize)size completionHandler:(WPMediaImageBl
     return [self.mediaID intValue];
 }
 
+- (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler
+{
+    if (!completionHandler) {
+        return 0;
+    }
+
+    NSString *errorMessage;
+    // Check if asset being used is a video, if not this method fails
+    if (self.assetType != MediaTypeVideo) {
+        errorMessage = NSLocalizedString(@"Media selected is not a video.",@"Error message when user tries to preview an image media like a video");
+        completionHandler(nil, [self errorWithMessage:errorMessage]);
+        return 0;
```

Comment:
Possibly outside of the scope of this PR, but should we have a constant that represents a non-existent request ID?